### PR TITLE
Local development linking

### PR DESCRIPTION
**Pull request by scottsdevelopment**
- Add commands for local development linking
- Update documentation on commands
- Remove version.json as it won't be used
- Add license to package json (inferred by vets-api and vets-website) to prevent warning spam

- department-of-veterans-affairs/va.gov-team#79995

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,9 +12,15 @@
     "test": "mocha 'test/**/*.spec.js'",
     "test:all-schemas": "mocha 'test/common/all-schemas.spec.js'",
     "watch": "node src/start watch",
+    "link": "yarn link:vets-api; yarn link:vets-website",
```
Comment (wayne-weibel) [inline package.json:15]: use a different name so there is no confusion with the builtin yarn command

File: package.json [hunk 2]
```diff
@@ -11,9 +12,15 @@
...
+    "link:vets-api": "cd ../vets-api; bundle config set local.vets_json_schema ../vets-json-schema; bundle config disable_local_branch_check true",
+    "link:vets-website": "yarn link; cd ../vets-website; yarn link vets-json-schema",
```
Comment (wayne-weibel) [inline package.json:17]: should move these to a shell script, to keep the package.json cleaner and can annotate the commands

**Review by wayne-weibel — CHANGES_REQUESTED** [review]
Please add a way to test that the local json-schema is being referenced ...

I am seeing an ‘error’ doing unlink and the script is not using the command noted in the output to reinstall the package; would also like this to be in a shell script so it it easier to read and can be annotated

image

**Review by wayne-weibel — APPROVED** [review]

**Review by RachalCassity — APPROVED** [review]

**Comment by pmclaren19** [comment]
@RachalCassity @scottsdevelopment @wayne-weibel trying to follow these instructions here and they are not working. Anyone free to pair?

**Pull request merged**